CMakeLists.txt

```diff
@@ -94,6 +94,12 @@ if(${FORCE_COLORED_OUTPUT})
   endif()
 endif()
 
+#Solves compile issue for debug builds
+#https://learn.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/fatal-error-c1128?view=msvc-170
+if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug" AND MSVC)
+  add_compile_options(/bigobj)
```

Comment:
We have this definition for amalgamation builds in `src/CMakeLists.txt:36` - perhaps we can just move it up one layer? I think we might need it for more than just debug builds, and it doesn't hurt to enable for all Windows builds